CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Tableau Connector SDK Changelog
+## 2021-09-02
+### Changed
+- Validate max length of `name` in Company-G in `connector_plugin_manifest_latest.xsd`
```

Comment:
Thank you for being the only person to remember to do this.

Reply:
Lol